metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/ElasticsearchIndexingIntegrationTest.java

```diff
@@ -102,11 +102,16 @@ public void setAdditionalProperties(Properties topologyProperties) {
     topologyProperties.setProperty("es.clustername", "metron");
     topologyProperties.setProperty("es.port", "9300");
     topologyProperties.setProperty("es.ip", "localhost");
-    topologyProperties.setProperty("indexing.writer.class.name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
+    topologyProperties.setProperty("indexing_writer_class_name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
   }
 
   @Override
   public String cleanField(String field) {
     return field;
   }
```

Comment:
I am not sure about having things live in ambari vs. things being referenced by ambari. That doesn't seem right, like a violation of a separation of concerns.  I know we do this other places too ( like the ES Templates, although I'm trying to move away from that ).   I am not sure I have a different way to suggest, but I want to bring up the point.